automation/taskcluster/androidTest/parse-ui-test.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+from __future__ import print_function
+
+import sys
+import argparse
+from pathlib import Path
+import json
+import yaml
+
+def parse_args(cmdln_args):
+    parser = argparse.ArgumentParser(description="Parse UI test logs an results")
+    parser.add_argument(
+        "--output-md",
+        type=argparse.FileType("w", encoding="utf-8"),
+        help="Output markdown file.",
+        required=True,
+    )
+    parser.add_argument(
+        "--log",
+        type=argparse.FileType("r", encoding="utf-8"),
+        help="Log output of flank.",
+        required=True,
+    )
+    parser.add_argument(
+        "--results", type=Path, help="Directory containing flank results", required=True
+    )
+    parser.add_argument(
+        "--exit-code", type=int, help="Exit code of flank.", required=True
```

Comment:
Thanks, Johan.  We'd definitely like the tests to be running on every PR as before #7617 (and same as fenix) but if they haven't been running since end of July, there's a risk of multiple tests having been broken in the meantime and they may block the build all at once when merging.  When should we re-enable these?... in this PR or in a separate one?  Just want to make sure we address any broken tests before re-activating them and then having broken tests blocking the build.

Reply:
Let's do it in a separate PR. I believe it won't be a trivial change.

Reply:
I looked into it further and it turns out it's unrelated to #7617. We haven't enabled these tests to run on PRs: https://github.com/mozilla-mobile/android-components/blame/936c1fa174c4fb48c111154c68244370b4ba8d39/taskcluster/ci/test/kind.yml#L33

I double-checked to see if a PR that predates #7617 had those tests but this one doesn't, for instance: https://github.com/mozilla-mobile/android-components/runs/918774631